website/cue/reference/releases/0.42.0.cue

```diff
@@ -0,0 +1,379 @@
+				"""
+		},
+		{
+			type: "enhancement"
+			description: """
+				Adds more aggregations to `aggregate` transform: count, diff, max, min, mean, sum, latest and stdev. Current aggregation is named `auto` and is made the default (acts like a combination of `sum` and `latest`).
+				"""
+			contributors: ["esensar"]
+		},
+		{
+			type: "enhancement"
+			description: """
+				Adds support for configuring `gcp_cloud_storage` sink API `endpoint`.
+				"""
+			contributors: ["movinfinex"]
+		},
+		{
+			type: "feat"
+			description: """
+				Adds support `OPTIONS` HTTP-method for `http_server` source
+				"""
+			contributors: ["sillent"]
+		},
+		{
+			type: "feat"
+			description: """
+				Adds support for additional `graph` configuration on each component so that users can add arbitrary graphviz node attributes when generating a graph via `vector graph`.
+				"""
+			contributors: ["esensar"]
+		},
+		{
+			type: "enhancement"
+			description: """
+				Expose a `retain` boolean config flag in the MQTT sink to tell the server to retain the messages.
+				"""
+			contributors: ["miquelruiz"]
+		},
+		{
+			type: "fix"
+			description: """
+				Fixed the `logstash` source decoder to remove a panic that could trigger in the event of certain protocol errors.
+				"""
+		},
+		{
+			type: "feat"
+			description: """
+				The `axiom` sink now uses the native HTTP API rather than the Elasticsearch-compatible API.
+
+				Notable changes:
+
+				- The elasticsearch `@timestamp` semantics that changed in v5.5 of ElasticSearch no
+				  longer affect the `axiom` sink as the sink uses the [Axiom native HTTP ingest
+				  method](https://axiom.co/docs/send-data/ingest#ingest-api).
+				- The `_time` field in data sent to Axiom now supports the same semantics as the
+				  Axiom native API and SDK [as
+				  documented](https://axiom.co/docs/reference/field-restrictions#requirements-of-the-timestamp-field).
+				  In previous versions of Vector, the Axiom sink rejected events with `_time`
+				  fields as the sink was following Elasticsearch semantics. This was confusing and
+				  suprising for seasoned Axiom users new to Vector and seasoned Vector users new
+				  to Axiom alike.
+				- If a `@timestamp` field is sent to Axiom it is a normal user defined field.
+				- If an `_time` field is sent to Axiom it now follows documented Axiom field semantics.
+				"""
+			contributors: ["darach"]
+		},
+		{
+			type: "fix"
+			description: """
+				Fixed a bug in the `new_relic` sink that caused dotted attribute names to be encoded incorrectly with quotes.
+				"""
+		},
+		{
+			type: "fix"
+			description: """
+				Previously, when the `new_relic` sink sent non-standard event fields to the logs
+				API, it would include those fields beside the standard event fields (i.e.
```

Comment:
```suggestion
				API, it would include those fields beside the standard event fields (for example, 
```

Reply:
I think `i.e.` should actually be expanded to `that is,` rather than `for example,`. I appreciate the call outs on expanding these abbreviations though!